### DISCUSSION TITLE
Repeat first line of layout--single

### DISCUSSION THREAD

**Discussion by JuanDCG314** [mmistakes/minimal-mistakes#3208] [Q&A]
Hi Rose,

I want to use your mmistakes theme to build some github pages.

For now, I've a problem:

Why the first line of a layout--single appears two times?

Examples: 
https://web.aulataller314.org/aulataller314/
https://web.aulataller314.org/autores/

What am I doing wrong??

Thanks very much.

**Comment by iBug** [comment]
There's no way we can diagnose unless you share a public repo for us to investigate.

**Reply by JuanDCG314** [reply]
Thanks @iBug , I fix it just now with layout--splash instead layout--single. But with single layout persist the problem... Look at "autores"

Sorry, this link are on public github pages with custom domains, so you can help me or investigate "my way" to customize mmistakes.

Organization that contains repositories are inside:
https://github.com/AulaTaller314

Thanks very much, maybe I should booking some help, but for now I'm investigating it.

I want to launch a documentation webs and above (web), is like the landing page of the others.

Thanks for your answer.

**Reply by iBug** [reply]
Can you please share a *specific* repository where your issue can be reproduced & verified? I took a quick glance but none of the repositories look like "autores".

As a general tip, you want to make it easier *for others to help you* when seeking help.

**Reply by iBug** [reply]
OK I found your page: <https://github.com/aulataller314/web/tree/main/_pages>

This is because GitHub Pages loads the [jekyll-titles-from-headings](https://github.com/benbalter/jekyll-titles-from-headings) plugin by default. As a result, the title is retrieved from the starting heading and displayed separately from the content.

To stop this from happening, you should stop starting the post content with a heading (i.e. `# Markdown Heading`), and specify the title in Front Matter:

```yaml
---
title: An Awesome Title
---

Main content
```

**Reply by JuanDCG314** [reply]
OK, thank you very much @iBug , good advise, I fix it, now, you can see it.

https://web.aulataller314.org/autores/
https://github.com/aulataller314/web

I work from my personal account and PR to the organization, but for now I'm alone.

Kind regards.
